### DISCUSSION TITLE
Tailwind Template Path Issue

### DISCUSSION THREAD

**Discussion by CreateSean** [solspace/craft-freeform#1639] [Questions & Answers] [answered]
### What happened?

I copied the tailwind templates (from the docs) to my template folder in this heirarchy

```
_freeform
  fields
    _default.twig
    checkbox.twig
    checkboxes.twig
    dropdown.twig
    group.twig
    opinion-scale..twig
    radios.twig
    table.twig
  formatting
    main.css
    main.js
    row.twig
  site.twig
```

I then updated the settings for the form to use the correct freeform template

![image](https://github.com/user-attachments/assets/cfe392dd-e55d-4aff-94ef-2fe41723677d)

When I view the form on the front end i get this error:

```
Invalid Argument – [yii\base\InvalidArgumentException](https://www.yiiframework.com/doc-2.0/yii-base-invalidargumentexception.html)
The file or directory to be published does not exist: /_freeform/formatting/_main.css
```

I then updated the paths for _main.css and _main.js to match what I have and it still gives me this error:

```
{# CSS overrides #}
{% set cssPath = view.assetManager.publishedUrl('_freeform/formatting/_main.css', true) %}
{% do view.registerCssFile(cssPath) %}

{# JS overrides #}
{% set jsPath = view.assetManager.publishedUrl('_freeform/formatting/_main.js', true) %}
{% do view.registerJsFile(jsPath) %}

{% import "_freeform/formatting/_row.twig" as rowMacro %}
```

### How can we reproduce this?

1. update to freeform 5.7.1
2. change formatting template from bootstrap 3 to copied tailwind templates
3. load form 

### Freeform Edition

Lite

### Freeform Version

5.7.1

### Craft Version

4.12.8

### When did this issue start?

After upgrading from older Freeform version

### Previous Freeform Version

highest version 4

**Comment by kjmartens** [comment, marked as answer]
Hi @CreateSean,

In this case, you should place those embedded templates within a new folder, e.g. `site`, and place `site.twig` inside of it as well, and rename it to `index.twig`. 

For example:

```
_freeform/
  site/
    index.twig
    fields/
      _default.twig
      checkbox.twig
      checkboxes.twig
      dropdown.twig
      group.twig
      opinion-scale..twig
      radios.twig
      table.twig
    formatting/
      main.css
      main.js
      row.twig
```

Please let me know if this resolves the issue for you. 🙂

**Reply by CreateSean** [reply]
Kelsey,

Yes that fixes it. Thank you.
